tests/all/wast.rs

```diff
@@ -26,11 +38,12 @@ fn run_wast(wast: &str, strategy: Strategy) -> anyhow::Result<()> {
         .wasm_reference_types(reftypes)
         .wasm_multi_value(multi_val)
         .strategy(strategy)?
-        .cranelift_debug_verifier(true);
+        .cranelift_debug_verifier(cfg!(debug_assertions));
```

Comment:
Will this disable the verifier for fuzzing?

Reply:
Great point, although I'm not sure if it actually will disable these checks in fuzzing - this file appears to only be for the spec test suite, and fuzzing should really have `debug_assertions` turned on anyway. Either way, I can easily revert this change, as I made it before I split the benchmarks into their own file - before, the tests and benchmarks were identical except that the benchmarks had `b.iter(|| {})` around `wast_context.run_file(wast)`. It's now not really necessary.